content/en/docs/setup/upgrade/istioctl-upgrade/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Upgrade and downgrade Istio using istioctl [Experimental]
```

Comment:
Since downgrade is also done with the same `istioctl x upgrade ...` command, I wouldn't call it downgrade. I would just suggest that in the document you point out that istioctl upgrade can be used to change an earlier version too, and then explain the extra considerations.

```suggestion
title: Upgrade using istioctl [Experimental]
```

Reply:
I agree with frankbu that we'd better not mention downgrade in official documentation. This is also quite common in other softwares such as Kubernetes: https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/.